README.md

```diff
@@ -0,0 +1,59 @@
+# Django - React/Vite.js Cookiecutter template
+
+## Overview
+
+This template is used for web development project that uses Django and React + Vite.js
+
+## Quickstart
+
+Run cookiecutter against this repo's url:  
+```python 
+cookiecutter https://github.com/shiyingwucl/cookiecutter 
+```
+
+This will prompt the following configurations for your project:
```

Comment:
Maybe add that if they don't set a value themselves, the one previewed in brackets will be selected/set?